### PR TITLE
Change cli plugin release flow

### PR DESCRIPTION
**Pull request by gailazar300**
Upload plugin's binaries and distribute to releases.jfrog.io.

### CODE REVIEW

**Comment by gailazar300** [comment]
I have read the Developer Terms and I hereby accept the terms

File: pipelinesScripts/pluginUpload.sh [hunk 1]
```diff
@@ -86,7 +75,7 @@ buildAndUpload () {
   destPath="$JFROG_CLI_PLUGINS_RT_REGISTRY_REPO/$JFROG_CLI_PLUGIN_PLUGIN_NAME/$JFROG_CLI_PLUGIN_VERSION/$pkg/$exeName"
   echo "Uploading $exeName to $JFROG_CLI_PLUGINS_RT_REGISTRY_URL/$destPath ..."
 
-  ./jfrog rt u "./$exeName" "$destPath" --url="$JFROG_CLI_PLUGINS_RT_REGISTRY_URL" --access-token=$JFROG_CLI_PLUGINS_RT_REGISTRY_TOKEN
+  res=$(eval "./jfrog rt u ./$exeName $destPath")
```
Comment (RobiNino) [inline pipelinesScripts/pluginUpload.sh:78]: The result of the command doesn't seems to be used, any reason to insert into this eval?

File: pipelinesScripts/pluginUpload.sh [hunk 2]
```diff
@@ -99,7 +88,7 @@ copyToLatestDir () {
   pluginPath="$JFROG_CLI_PLUGINS_RT_REGISTRY_REPO/$JFROG_CLI_PLUGIN_PLUGIN_NAME"
   echo "Copy version to latest dir: $pluginPath"
 
-  ./jfrog rt cp "$pluginPath/$JFROG_CLI_PLUGIN_VERSION/(*)" "$pluginPath/latest/{1}" --flat --url="$JFROG_CLI_PLUGINS_RT_REGISTRY_URL" --access-token=$JFROG_CLI_PLUGINS_RT_REGISTRY_TOKEN
+  res=$(eval "./jfrog rt cp \"$pluginPath/$JFROG_CLI_PLUGIN_VERSION/(*)\" \"$pluginPath/latest/{1}\" --flat")
```
Comment (RobiNino) [inline pipelinesScripts/pluginUpload.sh:91]: Is this copy still needed if the plugin are being distributed to releases and downloaded from there?

If it is:
The result of the command doesn't seems to be used, any reason to insert into this eval?

**Review by RobiNino — CHANGES_REQUESTED** [review]

File: pipelines.yml [hunk 3]
```diff
@@ -53,46 +53,56 @@ pipelines:
...
+                tag: 1.4.0
           environmentVariables:
             JFROG_CLI_PLUGIN_DESCRIPTOR_FILE_NAME: "file-name-with-extension.yml"
         execution:
           onStart:
             - update_commit_status buildJfrogCliPluginGit --context "$step_name"
           onExecute:
             # Install yq to parse yaml.
-            - GO111MODULE=on go get github.com/mikefarah/yq/v3
+            - GO111MODULE=on go get github.com/mikefarah/yq/v4
```
Comment (RobiNino) [inline pipelines.yml:74]: go get: installing executables with 'go get' in module mode is deprecated.
	Use 'go install pkg@version' instead.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.

**Review by RobiNino — CHANGES_REQUESTED** [review]

File: pipelines.yml [hunk 4]
```diff
@@ -80,18 +81,27 @@ pipelines:
             - pluginRelativePath=$(~/go/bin/yq e ".relativePath // \"\"" $ymlPath) && echo $pluginRelativePath
             # Clone plugin repo.
             - git clone $pluginRepoFullUrl
-            # Copy the plugin release script from registry repo to plugin repo.
-            - cp "$res_buildJfrogCliPluginGit_resourcePath/pipelinesScripts/pluginRelease.sh" "./$pluginRepoName/$pluginRelativePath"
+            # Copy the plugin release script and distribution spec from registry repo to plugin repo.
+            - cp "$res_buildJfrogCliPluginGit_resourcePath/pipelinesScripts/pluginUpload.sh" "./$pluginRepoName/$pluginRelativePath"
+            - cp "$res_buildJfrogCliPluginGit_resourcePath/pipelinesScripts/specs/plugin-rbc-spec.json" "./$pluginRepoName/$pluginRelativePath"
             - cd "./$pluginRepoName/$pluginRelativePath"
+            # Download JFrog CLI
```
Comment (eyalbe4) [inline pipelines.yml:88]: Download --> Install

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

**Pull request merged**